recipes-bsp/tegra-binaries/tegra-helper-scripts-native_34.1.0.bb

```diff
@@ -7,7 +7,7 @@ INHIBIT_DEFAULT_DEPS = "1"
 inherit native
 
 SRC_URI = " \
-    file://tegra194-flash-helper.sh \
+    file://${SOC_FAMILY}-flash-helper.sh \
```

Comment:
Instead of this change, just add `tegra234-flash-helper.sh` to the SRC_URI.  Take a look at the version of this recipe in the `master` branch as an example - it installs all of the helper scripts for all chip variants.